packages/bpk-docs/src/pages/NeoNavigationStackPage/NavigationStackPage.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
+import DocsPageWrapper from './../../components/neo/DocsPageWrapper';
+import Paragraph from './../../components/Paragraph';
+
+import Web from '../NavigationStackPage';
+
+const Page = () => (
+  <DocsPageWrapper
+    title="Navigation stack"
+    blurb={[
+      <Paragraph>
+        The navigation stack component presents content in full-screen and makes
+        use of the navigation bar component to allow pagination between screens.
+      </Paragraph>,
+    ]}
+    webSubpage={<Web wrapped />}
+  />
+);
+
+export default Page;
```

Comment:
Because there's no native version of this, you don't need to have a `Neo` folder, you can just implement `DocsPageWrapper` inside the other folder. See https://github.com/Skyscanner/backpack/blob/master/packages/bpk-docs/src/pages/BlockquotesPage/BlockquotesPage.js#L61 for an example.